SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme

```diff
@@ -130,6 +130,10 @@
             argument = "xcassets -t swift3 $(PROJECT_DIR)/Tests/Fixtures/Resources/XCAssets/Images.xcassets"
             isEnabled = "NO">
          </CommandLineArgument>
+         <CommandLineArgument
+            argument = "yaml $(PROJECT_DIR)/Tests/Fixtures/Resources/Yaml/root-mapping.yaml"
```

Comment:
Ok. So that command just test that it shows the error of missing a template param then? (gtm)

We could imagine a specific YAML template to build structs from a YAML tree that would have to follow a strict convention. Or one that parse some well-known file format written in YAML, say swagger specs?